Vagrantfile

```diff
@@ -320,9 +320,7 @@ Vagrant.configure('2') do |config|
 
   # Configurations from 1.0.x can be placed in Vagrant 1.1.x specs like the following.
   config.vm.provider :virtualbox do |v|
-    # Move the ubuntu-bionic-18.04-cloudimg-console.log file to log directory.
-    v.customize ['modifyvm', :id, '--uartmode1', 'file', File.join(vagrant_dir, 'log/ubuntu-bionic-18.04-cloudimg-console.log')]
-
+    v.customize ['modifyvm', :id, '--uartmode1', 'disconnected']
```

Comment:
This line causes VirtualBox and Vagrant to lock up. Power-cycling my machine is the only way I figured out to recover